app/services/data_requests/lookup_user_by_uuid.rb

```diff
@@ -7,8 +7,9 @@ def initialize(uuid)
     end
 
     def call
+      # mattw: This seems like a confusing use case. Used only in data_requests.rake FWIW.
       User.find_by(uuid: uuid) ||
-        ServiceProviderIdentity.find_by(uuid: uuid)&.user
+        ServiceProviderIdentity.consented.find_by(uuid: uuid)&.user
```

Comment:
im thinking maybe we take this out for better reporting, because the agency already has the UUID